examples/simple_example/main.tf

```diff
@@ -26,4 +26,7 @@ module "cli" {
 
   create_cmd_body  = "services enable youtube.googleapis.com --project ${var.project_id}"
   destroy_cmd_body = "services disable youtube.googleapis.com --project ${var.project_id}"
+
+  create_script  = "${path.module}/scripts/script.sh"
```

Comment:
We should make a script a separate example from a simple command (though I'm actually curious as to whether scripts could simply be passed as a command?)

Reply:
I've added a `script_example` and test. Similar to the `simple_example` but using a script to enable and disable an api. 

I could probably remove the new `script` vars in favor of utilizing the existing `create|destroy_cmd_entrypoint` variables. 

Right now they default to `gcloud` - but passing in the script instead should work (assuming I also prepend the bin directory to the path). Arguments to the script could use the existing `create|destroy_cmd_body` vars. 

Something like this uses all existing vars and will achieve the same effect without breaking original functionality. 

```hcl
module "cli" {
  source = "../.."

  platform              = "linux"
  additional_components = ["kubectl", "beta"]

  create_cmd_entrypoint = "${path.module}/scripts/script.sh"
  create_cmd_body       = "enable ${var.project_id}"

  destroy_cmd_entrypoint = "${path.module}/scripts/script.sh"
  destroy_cmd_body       = "disable ${var.project_id}"
}

```

Reply:
I went ahead and committed that. It does make the change quite a bit simpler. I'll update the `project-factory` PR for this approach too